development/build/index.js

```diff
@@ -268,7 +268,12 @@ async function defineAndRunBuildTasks() {
     composeSeries(
       clean,
       styleTasks.prod,
-      composeParallel(scriptTasks.dist, staticTasks.prod, manifestTasks.prod),
+      composeParallel(
+        scriptTasks.dist,
+        staticTasks.prod,
+        manifestTasks.prod,
+        manifestTasks.scriptDist,
```

Comment:
It looks like we're generating the manifest twice in parallel here. That's not good.

Could you elaborate on what the intention behind this `scriptDist` task was? What type of build is this for, when should it run?

Reply:
Note that `manifestTasks.prod` is still here, and `prepPlatforms` is part of both tasks. Maybe we can just remove that in favor of a `dist` task, rather than combining them?